pkg/translator/prometheusremotewrite/number_data_points.go

```diff
@@ -78,24 +79,26 @@ func addSingleSumNumberDataPoint(
 	}
 	sig := addSample(series, sample, labels, metric.Type().String())
 
-	if ts, ok := series[sig]; sig != "" && ok {
+	if ts := series[sig]; sig != "" && ts != nil {
```

Comment:
As commented above I was always under the impression that using the `ok` was common practice. Is there a net gain in doing it this way?

Reply:
Same answer as above. It's for robustness, guarding yourself against the case where `ts` might be nil. Checking whether `ts` is non-nil is a superset of checking whether `ok` is true (if `ts` is non-nil, `ok` has got to be true).